.github/workflows/ghci.yml

```diff
@@ -269,6 +269,9 @@ jobs:
         if: matrix.skip_tests != 'yes'
         run: |
           ART_REG_SKIP="${{ matrix.art_reg_skip }}" make -j 1 check
+          RET=$?
+          cat test-suite.log
+          exit $RET
 
```

Comment:
i suspect this won’t work since GHA usually exits on the first non-success; this should be better:
```sh
ART_REG_SKIP="${{ matrix.art_reg_skip }}" make -j 1 check || (cat test-suite.log; exit 1;)
```

Reply:
Oh, I'd gotten `set -e` and `set +e` mixed up, I guess it's -e by default. This looks reasonable.